test/Microsoft.DotNet.Cli.Utils.Tests/GivenAProjectToolsCommandResolver.cs

```diff
@@ -260,16 +261,64 @@ public void Generate_deps_json_method_doesnt_overwrite_when_deps_file_already_ex
             File.Delete(depsJsonFile);
         }
 
-        private ProjectToolsCommandResolver SetupProjectToolsCommandResolver(
-            IPackagedCommandSpecFactory packagedCommandSpecFactory = null)
+        [Fact]
+        public void It_adds_fx_version_as_a_param_when_the_tool_has_the_prefercliruntime_file()
+        {
+            var projectToolsCommandResolver = SetupProjectToolsCommandResolver();
+
+            var testInstance = TestAssetsManager
```

Comment:
Why TestAssetsManager and not TestAssets? I'd love to start eliminating the old pattern...